pyproject.toml

```diff
@@ -0,0 +1,11 @@
+[tool.towncrier]
```

Comment:
Slimed down the configuration in #546 to use more defaults and used the existing types, but gave them new names